README.md

```diff
@@ -37,6 +43,19 @@ minikube tunnel # in a separate terminal, as it keeps running
 ./scripts/drogue.sh
 ~~~
 
+## Useful Links
+
+* [Documentation](https://book.drogue.io/drogue-cloud/dev/index.html) 
+* [Drogue IoT Blog: Articles that talk about the design,  usecases and project updates](https://blog.drogue.io/)
+
 ## Contributing
 
 See the document [CONTRIBUTING.md](CONTRIBUTING.md).
+
+## Community
+
+* [Drogue IoT IRC Chat Room](https://matrix.to/#/#drogue-iot:matrix.org)
```

Comment:
```suggestion
* [Drogue IoT Matrix Chat Room](https://matrix.to/#/#drogue-iot:matrix.org)
```

Reply:
Technically it is "matrix", not "IRC".

Reply:
Thanks for that!

Reply:
So would changing the 'x' into a "Coming Soon" suffice?